### DISCUSSION TITLE
Exposing Valley Bottom Centerlines in VBET

### DISCUSSION THREAD

**Discussion by shelbysawyer** [Riverscapes/RiverscapesXML#140] [Q&A] [answered]
Right now, the valley bottom centerlines are buried in the centerlines output geopackage and aren't visible in VBET projects. I've got two questions here:

1. I'm not sure which of the centerline layers is the one that we want to show.. I see quite a few. From the [video](https://www.youtube.com/watch?v=pxwapXjZSMA) that @joewheaton made, it sounds like @KellyMWhitehead might have the answer to this.

2. How do I go about exposing this in the business logic? @joewheaton Maybe we can cover this quickly when we discuss the issue I'm having with symbolizing repeater layers in the business logic.

I did add in a few of the 'clean' centerlines in different colors just to see how they are being drawn in Arc currently:
<img width="264" alt="centerlines" src="https://user-images.githubusercontent.com/73311751/119907287-30415480-bf0d-11eb-80b1-317968c51d38.PNG">
<img width="1021" alt="centerlines1" src="https://user-images.githubusercontent.com/73311751/119907291-30d9eb00-bf0d-11eb-82bf-4ee51d8e5df3.PNG">

**Comment by KellyMWhitehead** [comment]
Please ignore everything in the `centerlines` folder - not sure how that got added to the project. 

There should only be one centerlines layer: `outputs/vbet.gpkg/vbet_centerlines_50`

**Comment by shelbysawyer** [comment]
Thanks @KellyMWhitehead!

I'm not seeing the centerlines layer that you're referencing:
<img width="307" alt="vbet" src="https://user-images.githubusercontent.com/73311751/119908201-4c45f580-bf0f-11eb-9cde-b4396cbd3d76.PNG">

Maybe this is an issue that's related to the other issues we noticed with this project? #79

**Comment by KellyMWhitehead** [comment, marked as answer]
@shelbysawyer please try [this project](https://data.riverscapes.xyz/#/PUBLIC_DATA/a40a1e2f-54f4-4da2-ae3c-feab6ff5973c). Let me know if you have any issues or questions with this version.

**Comment by joewheaton** [comment]
- [x] @shelbysawyer did the new project @KellyMWhitehead gave you make sense for the correct CL?
- [ ] @shelbysawyer , yes we can show you how to add a new line to business logic. I'm thinking just adding a new node (maybe like line 57 in [vbet.xml](https://github.com/Riverscapes/RiverscapesXML/blob/master/RaveBusinessLogic/VBET.xml) 
- [ ]

**Comment by shelbysawyer** [comment]
@joewheaton Yes, the new project displayed as @KellyMWhitehead Kelly described it should. The centerlines looked right. Thanks Kelly!

**Reply by joewheaton** [reply]
I moved this to the discussion because it was really just a question, and has nothing actionable in it.
